framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -2014,9 +2006,26 @@ public AnnotatedTypeMirror getDefaultAnnotations(Tree tree, AnnotatedTypeMirror
      * @param tree an AST node
      * @param type the type obtained from tree
      * @param iUseFlow whether to use information from dataflow analysis
+     * @deprecated use {@link #addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)} instead
      */
+    @Deprecated
     protected void addComputedTypeAnnotations(
             Tree tree, AnnotatedTypeMirror type, boolean iUseFlow) {
+        boolean oldUseflow = useFlow;
+        useFlow = iUseFlow;
+        addComputedTypeAnnotations(tree, type);
+        useFlow = oldUseflow;
+    }
+
+    /**
+     * Like {@link #addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)}. Overriding
```

Comment:
This Javadoc refers to itself. It needs a proper explanation.

Reply:
Also adapt the javadoc for the now deprecated function.

Reply:
> This Javadoc refers to itself. It needs a proper explanation.

The Javadoc is duplicated and I removed it.

> Also adapt the javadoc for the now deprecated function.

I already added `@deprecated in `addComputedTypeAnnotations( Tree tree, AnnotatedTypeMirror type, boolean iUseFlow)` 's Javadoc.